wiki/osu!_wiki_Contribution_Guide/tr.md

```diff
@@ -0,0 +1,106 @@
+# osu! wiki katkı rehberi
+
+osu! wiki'yi daha iyi hale getirmek için gösterdiğiniz ilgiye teşekkür ederiz! Bu hızlı başlangıç kılavuzu GitHub ortamında hiç çalışmamış, osu! wiki'nin katkı iş akışına alışkın olmayan yeni başlayanlar için hazırlanmıştır. Eğer GitHub kullanmakta deneyimliyseniz, bu veri havuzundaki içerik güncellemeleri için ortak feature-branch iş akışını takip edebilir ve bu rehberi atlayabilirsiniz.
+
+Eğer herhangi bir noktada kafanız karıştıysa ve/veya daha fazla yardıma ihtiyacınız varsa, [osu!dev Discord](https://discord.gg/ppy) sunucusuna (`#osu-wiki` kanalına) mesaj atmaktan çekinmeyin.
+
+## Başlarken
+
+### Kayıt olma
+
+1. Eğer yoksa, GitHub hesabı oluşturmak için [kayıt olun](https://github.com/join).
+2. [Giriş yapın](https://github.com/login).
+3. [Fork etme](#fork-etme) kısmına devam edin.
+
+### Fork etme
+
+1. [`osu-wiki` reposuna](https://github.com/ppy/osu-wiki) gidin.
+
+2. `Fork` tuşuna tıklayın.
+
+   ![](img/fork.jpg)
+
+3. Fork'unuza gitmek için `Fork` tuşuna yeniden tıklayın.
+
+4. Aşağıdakine benzer bir resim görmeniz `ppy/osu-wiki` reposunun bir fork'unu ortaya çıkardığınız ve şu anda fork'unuzda bulunduğunuz anlamına gelir.
+
+   ![](img/forked.jpg)
+
+5. [Fork'unuzu senkronlama](#forkunuzu-senkronlama) kısmına devam edin.
+
+### Fork'unuzu senkronlama
+
+1. `osu-wiki` reponuzun fork'una gidin.
+
+2. Dosyaların üzerinde, branch'ınızın `ppy:master` ile kıyaslandığında "behind" *(geride)*, "ahead" *(ileride)*, veya "even" *(eşit)* olduğunu belirten bir yazı bulunur. Eğer "behind" veya "ahead and behind" gibi bir yazıyla karşılaşırsanız, behind-commit'lerin sayısının bir önemi olmaksızın, branch'ınız güncel değildir.
+
+   ![](img/fork-even.jpg "İyi \(Commit yok ve güncel\)")
+
+   ![](img/fork-ahead.jpg "İyi \(Bunlar sizin commit'leriniz\)")
+
+   ![](img/fork-behind.jpg "Muhtemelen kötü \(Branch'ınız güncel değil\)")
+
+   ![](img/fork-ahead-behind.jpg "Muhtemelen kötü \(Branch'ınız sizin commit'lerinizle güncel değil\)")
+
+3. Eğer zaten değiştirilmiş bir dosyayı düzenlemeyecekseniz, bu pek bir sorun değil. Ancak ne kadar çok commit gerideyseniz, değiştirilmiş dosyalarla karşılaşma durumunuz bir o kadar yüksek.
+
+4. Bunu düzeltmek için, [Yaygın Sorunlar makalesinden Branch'ım güncel değil! kısmına](/wiki/owcg/Common_Issues#my-branch-is-out-of-date!) gözatın.
+
+5. 4. adımı tamamladığınızda, [Çevrimiçi ya da yerel düzenleme](#çevrimiçi-ya-da-yerel-düzenleme) kısmına devam edin.
```

Comment:
I think the markdown parser might get confused with that `4.`, or at least that's what GitHub is doing. I'll merge this in later if there are no other comments and see if the website gets confused too.

Reply:
```suggestion
5. 4\. adımı tamamladığınızda, [Çevrimiçi ya da yerel düzenleme](#çevrimiçi-ya-da-yerel-düzenleme) kısmına devam edin.
```
that actually is a valid way to make nested lists with no content at the top level